src/routes/user.routes.js

```
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const userController = require('../controllers/user.controller');
+const authenticate = require('../middlewares/authMiddleware');
+
+router.get('/profile', authenticate, userController.getProfile);
+router.put('/profile/name', authenticate, userController.updateName);
+router.put('/profile/password', authenticate, userController.updatePassword);
+router.put('/profile/email', authenticate, userController.updateEmail);
```

All profile routes use `authenticate`, which satisfies the access-control requirement for authenticated-only access. Consider adding request-body validation middleware (e.g. `express-validator`) on the `PUT` routes to enforce expected payload shapes (`name`, `oldPassword`/`newPassword`/`confirm`, `currentPassword`/`newEmail`/`confirmEmail`) and fail fast before reaching controllers.